### PR TITLE
Add an update script

### PR DESCRIPTION
**Pull request by vigoux**
This allows to repin a language by simply doing:
```
./scripts/update-lockfile.sh [mylang]
```

### CODE REVIEW

File: scripts/update-lockfile.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      if [ "$lang" != "$1" ]
+      then
+        printf "$lang,"
+      fi
+    done
+  fi
+}
+
+TO_IGNORE=$(make_ignored $1)
+
```
Comment (theHamsta) [inline scripts/update-lockfile.sh:17]: You can also use jq to select the string `jq "keys | select(. != $1) | join(\",\")"` ( command composed on mobile) https://stedolan.github.io/jq/manual/#select(boolean_expression)

Comment (theHamsta) [reply scripts/update-lockfile.sh:17]: But bash should be fine also

**Review by theHamsta — APPROVED** [review]

File: scripts/update-lockfile.sh (hunk 1 above)
Comment (vigoux) [reply scripts/update-lockfile.sh:17]: As you wish, I am fine with both, but I am definitely more at ease with sh than with jq :wink:

Comment (theHamsta) [reply scripts/update-lockfile.sh:17]: You could also add a new env variable for the lua script whatever works best for you.

Comment (vigoux) [reply scripts/update-lockfile.sh:17]: I am fine with keeping the old one, it is sufficient to have the ignore one, so I guess keeping it like this is good.

**Pull request merged**